docs/sections/calculations/errors.md

```diff
@@ -565,346 +230,116 @@ $ verdi calcjob outputcat <pk_number> | less
-
-```{code-block} ipython
+### Checking the inputs
 
-In [1]: failed_calculation = load_node(<PK>)
-   ...: restart_builder = failed_calculation.get_builder_restart()
-
-```
-
-Type `restart_builder` and press Enter to verify that all inputs have already been set to those that were used for the original calculation.
-Let's give the new calculation some more steps in the SCF cycle in order to let it converge:
-
-```{code-block} ipython
-
-In [2]: parameters = restart_builder.parameters.get_dict()
-   ...: parameters['ELECTRONS']['electron_maxstep'] = 80
-
-```
-
-The `aiida-quantumespresso` plugin supports restarting a calculation by setting the corresponding `restart_mode` and attaching the remote working directory of the previous calculations as the `parent_folder` input [^f3]:
-
-```{code-block} ipython
-
-In [3]: parameters['CONTROL']['restart_mode'] = 'restart'
-   ...: restart_builder.parent_folder = failed_calculation.outputs.remote_folder
-   ...: restart_builder.parameters = Dict(dict=parameters)
-
-```
-
-Note that we've created a new `Dict` node for the modifed parameters since the original input is stored in the database and immutable.
-
-Finally, let's label this calculation as a restarted one and submit the new calculation:
-
-```{code-block} ipython
-
-In [4]: from aiida.engine import submit
-   ...: restart_builder.metadata.label = 'Restart from PwCalculation<{}>'.format(failed_calculation.pk)
-   ...: calculation = submit(restart_builder)
-
-```
-
-Inspect the restarted calculation to verify that, this time, it completes successfully.
-You should see a "finished" status with exit code zero when running `verdi process list -a -p1`.
-
-## Calculation results
-
-The results of a calculation can be accessed directly from the calculation node using the following:
-
-```{code-block} console
-
-$ verdi calcjob res <IDENTIFIER>
-
-```
-
-which will print the output dictionary of the 'scalar' results parsed by AiiDA at the end of the calculation.
-Note that this is actually a shortcut for:
+To verify and/or further investigate any information gathered by looking at the logs and outputs, it is sometimes convenient to take a look at the inputs used.
+Similar to when we tried the dry run, we can also inspect the input files that AiiDA made to run the *actual* calculation.
+We can first check the content of the calculation folder by running:
 
 ```{code-block} console
 
-$ verdi data dict show <IDENTIFIER>
+$ verdi calcjob inputls <pk_number> -c
 
 ```
 
-where `IDENTIFIER` refers to the `Dict` node attached as an output of the calculation node, with link name `output_parameters`.
-By looking at the output of the command, what is the total energy (`energy`) of the calculation you have run?
-What are its units?
-
-Similarly to what you did for the calculation inputs, you can access the output files via the commands:
+where `<pk_number>` corresponds to the `PK` number of your calculation.
```

Comment:
```suggestion
where `<pk_number>` corresponds to the PK number of your calculation.
```

Reply:
Haha so you are basically [reviewing yourself now](https://github.com/aiidateam/aiida-tutorials/pull/353#discussion_r658742857) =P

I saw your [other comment](https://github.com/aiidateam/aiida-tutorials/pull/353#discussion_r660549454) with the explanation. Hey @mbercx should we add "PK vs `PK`" to #361 or do we open a new issue for this?

Reply:
Yeah - had a change of heart since going through some other sections. I think this makes the most sense.